ygen/ir.go

```diff
@@ -119,36 +131,66 @@ type LangMapperBase struct {
 	schematree *schemaTree
 }
 
-// SetEnumSet is used to supply a set of enumerated values to the
+// setEnumSet is used to supply a set of enumerated values to the
 // mapper such that leaves that have enumerated types can be looked up.
 //
 // NB: This method is a set-up method that the user does not need to be invoked
```

Comment:
Just reading this back `a set-up method that the user _does not_ need to be invoked`? Does this mean that the user doesn't need to invoke it, or that they don't care whether this is called? It's a bit confusing.

Reply:
Fixed up comment, PTAL if this makes sense now.